pset-functions.js

```diff
@@ -31,6 +31,21 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack (playerCardScore, dealerCardScore) {
+  if (playerCardScore > 21 && dealerCardScore > 21) {
+    return 0
+  } else if (playerCardScore > 21) {
+    return "Dealer wins"
+  } else if (dealerCardScore > 21 ) {
+    return "Player wins"
+  } else {
+    return Math.max(playerCardScore, dealerCardScore)
```

Comment:
Great use of if/else statements! Also, that is a good use of javascript's built in Math method to save yourself some lines of code.